physionet-django/templates/about/database_content.html

```diff
@@ -23,785 +23,41 @@ <h2>Overview</h2>
 <p>
 This page displays a curated list of databases in the PhysioNet archives. To find more databases on PhysioNet, <a href="{% url 'content_index' %}">search our resources</a>.</p>
 <ul>
-<li> <a href="#clinical">Clinical Databases</a>: Data from critical
-       care clinical settings that may include demographics, vital
-       sign measurements made at the bedside, laboratory test results,
-       procedures, medications, caregiver notes, images and imaging
-       reports, and mortality (both in and out of hospital).</li>
-<li><a href="#waveform">Waveform Databases</a>: High resolution continuous recordings of physiological signals. Waveform databases are organized according to their signal and annotation types:
-<ul>
-  <li> <a href="#multi">Multi-Parameter Databases</a>:
-     Available signals vary, but may include ECG, continuous invasive blood
-     pressure, respiration, oxygen saturation, and EEG, among others.</li>
-  <li> <a href="#ecg">ECG Databases</a>.  Also see
-     <a href="#multi">Multi-Parameter Databases</a>, most of which include
-     ECG signals.</li>
-  <li> <a href="#rr">Interbeat (RR) Interval Databases</a>:
-     These contain beat annotations obtained from ECG recordings, but the
-     ECG signals are not available.  Also see <a href="#ecg">ECG Databases</a>,
-     most of which include beat annotations in addition to the original ECG
-     signals.</li>
-  <li> <a href="#cv">Other Cardiovascular Databases</a></li>
-  <li> <a href="#gait">Gait and Balance Databases</a></li>
-  <li> <a href="#neuro">Neuroelectric and Myoelectric Databases</a>: EEG, EHG, and more.</li>
-  <li> <a href="#synthetic">Synthetic Data</a></li>
-</ul>
-<li> <a href="#image">Image Databases</a></li>
-<li> <a href="#miscellaneous">Miscellaneous Databases</a></li>
+<li> <a href="#open">Open Access</a>: Data projects with open access.</li>
```

Comment:
These descriptions don't add much (e.g. "Open access: Data projects with open access"). If we include descriptions, they should be useful. 

Perhaps we could add something like: 
- "Open access datasets are shared under permissive licenses that allow broad reuse."
- "Restricted access datasets require a Data Use Agreement to be signed." etc

Reply:
Do you have more information for each? Is there a reference I can look at for a better description? No exactly sure the differences

Reply:
As far as I know, the only description is in the "Access" page for new submissions, but the description there isn't particularly clear. We can chat, but briefly:

- "Open access" means the content is shared under an open access license (e.g. MIT License; Creative Commons licenses).
- "Restricted access" means that users must (1) have a PhysioNet account and (2) sign a DUA.
- "Credentialed access" means that users must (1) a PhysioNet account, (2) sign a DUA, and (3) complete the online CITI training.